### PR TITLE
TELCODOCS#2040: Multi-network policy support for bond CNI and IPVLAN

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.18

Issues:
- [TELCODOCS-2040](https://issues.redhat.com/browse/TELCODOCS-2040)
- [TELCODOCS-2035](https://issues.redhat.com/browse/TELCODOCS-2035)

Link to docs preview:
- [Configuring multi-network policy](https://84671--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-multi-network-policy.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
Hi @zeeke 
Could you please review this PR?
Thanks!

File: networking/multiple_networks/configuring-multi-network-policy.adoc [hunk 1]
```diff
@@ -6,7 +6,15 @@ include::_attributes/common-attributes.adoc[]
...
-As a cluster administrator, you can configure a multi-network policy for a Single-Root I/O Virtualization (SR-IOV), MAC Virtual Local Area Network (MacVLAN), or OVN-Kubernetes additional networks. MacVLAN additional networks are fully supported. Other types of additional networks, such as IP Virtual Local Area Network (IPVLAN), are not supported. 
+As a cluster administrator, you can configure a multi-network policy for any of the following network types:
+
+* Single-Root I/O Virtualization (SR-IOV)
+* MAC Virtual Local Area Network (MacVLAN)
+* IP Virtual Local Area Network (IPVLAN)
+* Bond Container Network Interface (CNI) over SR-IOV
+* OVN-Kubernetes additional networks
+
+MacVLAN additional networks are fully supported.
```
Comment (zeeke) [inline networking/multiple_networks/configuring-multi-network-policy.adoc:17]: Do we know what this statement exactly means? It should be the same as `MAC Virtual Local Area Network (MacVLAN)`.

Comment (zeeke) [reply networking/multiple_networks/configuring-multi-network-policy.adoc:17]: I could not defend a position if a user claims an explanation, so it's okay to remove it from my side.

Comment (sr1kar99) [reply networking/multiple_networks/configuring-multi-network-policy.adoc:17]: Is there a concept of partial support and full support for the other network types?
It is only in this case I think this statement makes sense.

Comment (zeeke) [reply networking/multiple_networks/configuring-multi-network-policy.adoc:17]: There is only one note about the SR-IOV networks, that is stated right after this paragraph:
https://github.com/openshift/openshift-docs/pull/84671/files#diff-39bf1f67d4adfc408401fdf1ee603afb080600e2dfa028993efa249aad639d8fR21

Comment (sr1kar99) [reply networking/multiple_networks/configuring-multi-network-policy.adoc:17]: Removed "MacVLAN additional networks are fully supported." 
Thanks!

**Comment by sr1kar99** [comment]
Hi @ajaggapa 
Could you please review this PR?
Thanks!

**Comment by ajaggapa** [comment]
@sr1kar99 

> You must specify an annotation with the name of the network attachment definition that defines the macvlan or SR-IOV additional network

Please modify this line as mentioned below. Since, this api is not specific to only macvlan and sriov networks anymore.

> You must specify an annotation with the name of the network attachment definition that defines the additional network

**Comment by sr1kar99** [comment]
> @sr1kar99
> 
> > You must specify an annotation with the name of the network attachment definition that defines the macvlan or SR-IOV additional network
> 
> Please modify this line as mentioned below. Since, this api is not specific to only macvlan and sriov networks anymore.
> 
> > You must specify an annotation with the name of the network attachment definition that defines the additional network

@ajaggapa Updated as per the suggestion.

**Review by lahinson — COMMENTED** [review]
lgtm

**Pull request merged**